frame/staking/src/lib.rs

```diff
@@ -112,11 +112,17 @@
 //! The **reward and slashing** procedure is the core of the Staking pallet, attempting to _embrace
 //! valid behavior_ while _punishing any misbehavior or lack of availability_.
 //!
-//! Rewards must be claimed for each era before it gets too old by `$HISTORY_DEPTH` using the
-//! `payout_stakers` call. Any account can call `payout_stakers`, which pays the reward to the
-//! validator as well as its nominators. Only the [`Config::MaxNominatorRewardedPerValidator`]
-//! biggest stakers can claim their reward. This is to limit the i/o cost to mutate storage for each
-//! nominator's account.
+//! Rewards must be claimed for each era before it gets too old by
+//! [`HistoryDepth`](`Config::HistoryDepth`) using the `payout_stakers` call. Any account can call
+//! `payout_stakers`, which pays the reward to the validator as well as its nominators. Only the
+//! [`Config::MaxExposurePageSize`] nominator rewards can be claimed in a single call. When the
+//! number of nominators exceeds [`Config::MaxExposurePageSize`], then the nominators are stored in
+//! multiple pages of [`Config::MaxExposurePageSize`], each with up to a maximum of
+//! [`Config::MaxExposurePageCount`] pages. To pay out all nominators, `payout_stakers` must be
+//! called once for each available page. In a scenario where the number of nominators N exceed M, where M
```

Comment:
Don't ask me how I know, but this and the next line exceed 100 chars :D

Reply:
🙈